src/nvme/nbft.h

```diff
@@ -1227,7 +1229,7 @@ struct nbft_info {
  *
  * Return: 0 on success, errno otherwise.
  */
-int nvme_nbft_read(struct nbft_info **nbft, const char *filename);
+int nvme_nbft_read(nvme_root_t r, struct nbft_info **nbft, const char *filename);
```

Comment:
Just passing in a `nvme_root_t` root object for printing the message is not the right thing to do. During review of this code we figured that this is not the right thing to do. That's why this argument is missing. 

We have the same problem for all non fabric functions that we don't have a `nvme_root_t` object and we might also want to print errors etc in those functions. 

The only way I see a way around this is to introduce a global variable containing the file descriptor to print to. Obviously, not totally happy with it, but what else can we do?